math/src/field/fields/fft_friendly/babybear.rs

```diff
@@ -0,0 +1,102 @@
+use crate::{
+    field::{
+        element::FieldElement,
+        fields::montgomery_backed_prime_fields::{IsModulus, MontgomeryBackendPrimeField},
+        //traits::IsFFTField,
+    },
+    unsigned_integer::element::UnsignedInteger,
+};
+
+pub type U64 = UnsignedInteger<1>;
```

Comment:
Please move this to `unsigned_integer/element.rs` ([here](https://github.com/lambdaclass/lambdaworks/blob/29af640a8e4e22301e7866981f156281d03ab51a/math/src/unsigned_integer/element.rs#L22-L24)). So all unsigned integer type aliases are together.

Reply:
This a nice idea